File: 3030.java
//* @author popop0p0popo
import java.util.*;
import java.io.*;

public class Main{
 public static void main(String[] args){
  Scanner scanner=new Scanner(new BufferedReader(new InputStreamReader(System.in)));
		int n=scanner.nextInt();
		int r,e,c;
		for (int i=0;i< n ;i++ ){
			r=scanner.nextInt();
			e=scanner.nextInt();
			c=scanner.nextInt();
			if (r< e-c){
				System.out.println("advertise");
			}
			else if (r==e-c){
				System.out.println("does not matter");
			}
			else{
				System.out.println("do not advertise");
			}
		}
	}
}

File: 3030.cc
#include <iostream>
#include <cstdio>
#include <cstring>
#include <string>
#include <math.h>
#include <algorithm>

using namespace std;

int main() {
  long long int test, a, b, c;

  cin >> test;

  for (int i = 0; i < test; i++) {
    cin >> a >> b >> c;

    if (b - c > a)
      cout << "advertise" << endl;
    else if (b - c == a)
      cout << "does not matter" << endl;

    else
      cout << "do not advertise" << endl;
  }
  return 0;
}
